app/javascript/vue/my_modules/list.vue

```diff
@@ -0,0 +1,337 @@
+        headerName: this.i18n.t('experiments.table.column.status_html'),
+        sortable: false,
+        cellRenderer: this.statusRenderer
+      }
+    ];
+
+    if (this.currentViewMode === 'archived') {
+      columns.push({
+        field: 'archived_on',
+        headerName: this.i18n.t('experiments.table.column.archived_html'),
+        sortable: true
+      });
+    }
+
+    columns.push({
+      field: 'assigned',
+      headerName: this.i18n.t('experiments.table.column.assigned_html'),
+      sortable: false,
+      cellRenderer: AssignedUsers
+    });
+    columns.push({
+      field: 'tags',
+      headerName: this.i18n.t('experiments.table.column.tags_html'),
+      sortable: false,
+      cellRenderer: TagsRenderer
+    });
+    columns.push({
+      field: 'comments',
+      headerName: this.i18n.t('experiments.table.column.comments_html'),
+      sortable: false,
+      cellRenderer: CommentsRenderer,
+      notSelectable: true
+    });
+
+    const filters = [
+      {
+        key: 'query',
+        type: 'Text'
+      },
+      {
+        key: 'due_date',
+        type: 'DateRange',
+        label: this.i18n.t('experiments.table.filters.due_date')
+      }
+    ];
+
+    if (this.currentViewMode === 'archived') {
+      filters.push({
+        key: 'archived_at',
+        type: 'DateRange',
+        label: this.i18n.t('filters_modal.archived_on.label')
+      });
+    }
+
+    filters.push({
+      key: 'assigned_users',
+      type: 'Select',
+      optionsUrl: this.usersFilterUrl,
+      optionRenderer: this.usersFilterRenderer,
+      labelRenderer: this.usersFilterRenderer,
+      label: this.i18n.t('experiments.table.filters.assigned'),
+      placeholder: this.i18n.t('experiments.table.filters.assigned_placeholder')
+    });
+
+    filters.push({
+      key: 'statuses',
+      type: 'Select',
+      options: this.statusesList,
+      label: this.i18n.t('experiments.table.filters.status'),
+      placeholder: this.i18n.t('experiments.table.filters.status_placeholder')
+    });
+
+    this.columnDefs = columns;
+    this.filters = filters;
+  },
+  computed: {
+    viewRenders() {
+      const canvasLabel = this.currentViewMode === 'active'
+        ? this.i18n.t('toolbar.canvas_view') : this.i18n.t('toolbar.cards_view');
+      return [
+        { type: 'table' },
+        {
+          type: 'custom',
+          name: canvasLabel,
+          url: this.canvasUrl
+        }
+      ];
+    },
+    toolbarActions() {
+      const left = [];
+
+      if (this.createUrl && this.currentViewMode !== 'archived') {
+        left.push({
+          name: 'create',
+          icon: 'sn-icon sn-icon-new-task',
+          label: this.i18n.t('experiments.table.toolbar.new'),
+          type: 'emit',
+          path: this.createUrl,
+          buttonStyle: 'btn btn-primary'
+        });
+      }
+
+      return {
+        left,
+        right: []
+      };
+    }
+  },
+  methods: {
+    usersFilterRenderer(option) {
```

Comment:
would be nicer to group the renderer functions together, maybe at the end of the methods object